app/models/concerns/profile_policy_association.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# Methods that are related to a profile's policy
+module ProfilePolicyAssociation
```

Comment:
Rubocop was complaining about the length of `profile.rb`

Reply:
:+1: it's definitely tidier